projects/seronet-spawner/controllers/box_creator/box_creator.cc

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
We normally use the `.cpp` extension for C++ file, not `.cc`

Reply:
> We usually use the `.cpp` and `.hpp` file extensions for C++.
> However if these files were simply copied / pasted from somewhere else, maybe we can keep them as is.

The files in SeRoNet use `.cc` and `.hh`, but only `CoordinationRESTServer` was copied from there, so I think we can change the extensions...